supporter-product-data/src/main/scala/com/gu/model/zuora/response/MinimalZuoraSubscription.scala

```diff
@@ -0,0 +1,28 @@
+package com.gu.model.zuora.response
+
+import com.gu.supporterdata.model.ContributionAmount
+import io.circe.Decoder
+import io.circe.generic.semiauto.deriveDecoder
+
+case class MinimalZuoraSubscription(ratePlans: List[RatePlan]) {
+  def contributionAmount = for {
+    ratePlans <- ratePlans.headOption
+    charges <- ratePlans.ratePlanCharges.headOption
```

Comment:
we are assuming exactly one charge and plan - how about if there are multiple? can we return an Either and complain about it as clearly either the sub is broken, or someone has reused this code in future and it might quietly break.